vars/RedHat.yml

```diff
@@ -28,3 +28,6 @@ __php_opcache_conf_filename: 10-opcache.ini
 __php_fpm_daemon: php-fpm
 __php_fpm_conf_path: "/etc/fpm"
 __php_fpm_pool_conf_path: "/etc/php-fpm.d/www.conf"
+
+__php_fpm_pool_user: apache
```

Comment:
The other condition might be the ownership of the session directory ... something like `/var/lib/php/5.6/` ... it has to be server writeable or any application using sessions (which is pretty much every application these days) will fail.  File storage for sessions is the default in PHP.